coralproject/talk
# [CORL-2156]: sanitize content in message box config

## What does this PR do?

This ensures that we are using DOMPurify to sanitize the content entered in the message box config for a story by a moderator or admin.

## What changes to the GraphQL/Database Schema does this PR introduce?

none

## Does this PR introduce any new environment variables or feature flags? 

no

## If any indexes were added, were they added to `INDEXES.md`?

n/a

## How do I test this PR?

As a moderator or admin, go to `Configure` tab for a story. Add a message in the `Write a message` section. See that your message is shown, previewed, and saves as normal. Enter something that should be sanitized out by DOMPurify in the message box. See that it is not shown, previewed, or saved as you entered it.
 
## How do we deploy this PR?
